ng5/src/app/screens/container.scss

```diff
@@ -1,6 +1,6 @@
-@import '../../../sass/colors.scss';
```

Comment:
the file location is odd. If it is a global styles file it hould be in the sass folder